src/js/parsers/post.js

```diff
@@ -52,7 +52,7 @@ export default {
 
     // walk up from the textNode until the rootNode, converting each
     // parentNode into a markup
-    function collectMarkups(textNode, rootNode) {
+    const  collectMarkups = (textNode, rootNode) =>{
```

Comment:
using `const foo = () => {` is pretty harmless, we should wander and discuss if it should be used for basically any function though. I'm unclear why some are migrated to the new style and others are not, or why the new style is superior.